recipes/pulsar_phase/pulsar_phase_computation.ipynb

```diff
@@ -0,0 +1,1895 @@
+{
```

Comment:
<p>Is it useful to be in DEBUG mode for pint?</p><p>Should we note show how to handle the level of the PINT logger? And maybe put it as just INFO for this nb?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/gammapy/gammapy-recipes/pull/12/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='34'/>

Reply:
I mean this is the format of ephemeris file, so is it really our job to explain that ? 

We could put a link to the documentation pdf of Tempo2, which kind of explain all this. But isn't it going to be confusing to refer to Tempo2 while using PINT ...

Reply:
Sorry, replied to the wrong comment ...

Reply:
<p>It's the default log level of PINT, but we can probably change it. But it still gives interesting information about what it is going so I am not sure that we should deactivate it. </p>

Reply:
<p>I agree, some of them gives relevant information for the user so we should keep it.</p>